.github/workflows/deploy-book-ghpages.yml

```diff
@@ -72,6 +72,12 @@ jobs:
     - if: ${{ steps.cache-html.outputs.cache-hit != 'true' }}
       name: Preprocess & build the book from branch
       run: |
+        echo $PATH
+        pip install --user teachbooks --extra-index-url https://gitlab.tudelft.nl/api/v4/projects/13957/packages/pypi/simple
+        cat book/_toc.yml
+        teachbooks build --publish --process-only book/
+        mv book/.psorbooks/publish/_toc.yml book/.psorbooks/publish/_config.yml book/
```

Comment:
@CasparJungbacker looks like there is a still a reference to `.psorbooks` in the package

Reply:
@CasparJungbacker I added you as reviewer for this PR to check that there aren't other alignment issues to take care of with the teachbooks package. Spin them off into issues as needed to come back to later.

Reply:
#13

Reply:
Interesting. @severinbratus why did you choose to do `teachbooks build` with the option `--process-only`, then copy the processed files manually? Did you encounter any issues with teachbooks also building the book?

Reply:
@CasparJungbacker This was a mixup on my part. No, no issues encountered iirc. Will fix